cmd/insolard/main.go

```diff
@@ -39,7 +39,11 @@ type inputParams struct {
 }
 
 func parseInputParams() inputParams {
-	var rootCmd = &cobra.Command{Use: "insolard"}
```

Comment:
please put TODO here, so we 

1. either go with cobra command (put start into Run)
2. remove cobra (because config reading could be done without cobra)

It looks ugly as hell :(